### ISSUE TITLE
Submit buttons showing ❌ on validation errors

### ISSUE THREAD

**Issue by parkerdavis1** [opportunity-hack/evs#9]
When logging in, if username/password is not correct, the Login button shows an ❌ emoji which persists after subsequently editing the username and password fields. This is visually confusing because the ❌ implies the `Log in` button is disabled when it is not.

![image](https://github.com/opportunity-hack/evs/assets/92223355/0d219901-253b-4534-8814-9bb3a0d09cb4)

The same thing happens when signing up and you enter an email that already has an account. I'm sure this happens on many different buttons throughout the app. 

![image](https://github.com/opportunity-hack/evs/assets/92223355/09091fa2-ce78-4503-9edf-94b2841f1993)

I would suggest removing the ❌ on buttons altogether.

**Comment by benhsm** [comment]
I'm not sure I agree with you on this one - the X is meant as feedback indicating that the action associated with the button has failed, where color and cursor shape on hover are used to indicate whether the button is enabled. In any case, thanks for raising the issue; I'm not going to make any changes to this component for the time being, but this is open to evaluation.

**Comment by parkerdavis1** [comment]
Exactly, the X indicates there is a problem associated with the button, when the problem is really with the input field(s). Alternative feedback could be red outlines on the offending input fields or an alert box with the error message (invalid username or password).

**Comment by gregv** [comment]
Was able to repro this on https://staging.trottrack.org/signup and login, we want the X to go away on the button since it also is reset with the red outline on the text boxes.
